src/test/java/com/loofah/graph/api/repositories/SkillsRepositoryTest.java

```diff
@@ -0,0 +1,43 @@
+package com.loofah.graph.api.repositories;
+
+import com.loofah.graph.api.models.Skill;
+import com.loofah.graph.api.testConfig.GraphAPITestConfiguration;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
```

Comment:
I think that because you've used constructor injection for the `SkillsRespository`, we could just pass in a list of skills for this test rather than bringing in `SpringRunner` so we can autowire the skills in, and the test would run faster. What do you think?

Reply:
Yes you're right, this was just me playing around with autowiring but I've taken out the test config and spring runner and these tests run much faster now